news/templates/news/list.html

```diff
@@ -76,4 +76,30 @@
 		{% translate "ADD_NEWS_ITEM" %}
 	</a>
 </p>
+
+<script>
+window.addEventListener('DOMContentLoaded', function() {
+	var removeBtns = document.querySelectorAll("[id^='remove-btn']");
+
+	for (var i = 0; i < removeBtns.length; i++) {
+		removeBtns[i].addEventListener('click', function(event) {
+			event.preventDefault();
+
+			var currentUrl = window.location.href;
+			var newsId = this.getAttribute('value');
```

Comment:
Only use `var` if it is needed, I think `const` and `let` should be enough in this code.

Reply:
Everything is now const except the interator in the for loop which is now let